target/linux/generic/hack-6.12/931-mips-ralink-add-missing-include.patch

```diff
@@ -0,0 +1,17 @@
+From b71ab42067204c3d90aa9143b815482b222fee32 Mon Sep 17 00:00:00 2001
+From: Mieczyslaw Nalewaj <namiltd@yahoo.com>
+Date: Mon, 18 Nov 2024 21:28:05 +0100
+Subject: mips: ralink: add missing include
+
+Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>
+---
+--- a/arch/mips/ralink/irq.c
++++ b/arch/mips/ralink/irq.c
+@@ -15,6 +15,7 @@
+ 
+ #include <asm/irq_cpu.h>
+ #include <asm/mipsregs.h>
++#include <asm/time.h>
```

Comment:
Why not backport the commits I linked to?

Reply:
hrm nvm. they're already present.

Reply:
Ah I see why upstream didn't hit this. It depends on CONFIG_IRQ_INTC, which is not the case for mt7621.

Reply:
It was about another file.